html/patterns/01-atoms-08-something-else-something-else/01-atoms-08-something-else-something-else.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
Is "something else" a pattern you added? Was it a test? If so, please remove it and work on the patterns in the issue queue.